lib/beaker-pe/install/pe_utils.rb

```diff
@@ -450,7 +450,15 @@ def do_install hosts, opts = {}
               else
                 prepare_host_installer_options(host)
                 register_feature_flags!(opts)
-                generate_installer_conf_file_for(host, hosts, opts)
+
+                if opts[:upgrade] && use_meep?(host['previous_pe_ver'])
+                  # merge answers into pe.conf
+                  update_pe_conf(opts[:answers])
+                  update_pe_conf(opts[:custom_answers])
```

Comment:
discussed this morning, and determined that we should sync the `conf.d` dir with beaker since that is the currently documented process, and work on syncing automatically as a stretch goal.